merkletree/pad_test.go

```diff
@@ -235,3 +239,194 @@ func TestPoliciesChange(t *testing.T) {
 		t.Error(key3, "value mismatch")
 	}
 }
+
+func TestTB(t *testing.T) {
+	key1 := "key"
+	val1 := []byte("value")
+
+	pad, err := NewPAD(NewPolicies(3, vrfPrivKey1), signKey, 3)
+	if err != nil {
+		t.Fatal(err)
+	}
+	tb, err := pad.TB(key1, val1)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// TODO shouldn't there be a serialize function?
```

Comment:
Can we use this function? https://github.com/coniks-sys/coniks-go/blob/master/merkletree/tb.go#L21-L27